serverless/puppeteer-screenshots/serverless.yml

```diff
@@ -0,0 +1,40 @@
+service: puppeteer-screenshots
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+  memorySize: 1536
```

Comment:
Let's bump the memory up to max - although all we can control is the memory available, it does in fact change the EC2 instance behind the scenes to a more powerful one, giving us a faster CPU. It could improve the performance drastically, and serverless with low use is ~free, regardless of how much memory we use.